### ISSUE TITLE
Dropdown aria-labelledby incorrect

### ISSUE THREAD

**Issue by RomeroMsk** [bootstrap-vue/bootstrap-vue#1715]
Please check here: https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/src/components/dropdown/dropdown.js#L65

```js
'aria-labelledby': this.safeId(this.split ? '_BV_toggle_' : '_BV_button_')
```

Ternary expressions should be swapped if I understand correctly.

**Comment by emanuelmutschlechner** [comment]
Good catch. Requested a fix #1761

**Comment by emanuelmutschlechner** [comment]
@Diaskhan issue can be closed. fix was merged
